homeassistant/components/actronair_neo/coordinator.py

```diff
@@ -0,0 +1,154 @@
+"""Coordinator for Actron Air Neo integration."""
+
+from datetime import timedelta
+import logging
+import re
+
+import aiohttp
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ActronNeoDataUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class ActronNeoDataUpdateCoordinator(DataUpdateCoordinator[..]):
```
Please add the return type of `_async_update_data` here